scripts/terraform-docs.awk

```diff
@@ -0,0 +1,90 @@
+# This script converts Terraform 0.12 variables/outputs to something suitable for `terraform-docs`
```

Comment:
Let's use the upstream project in p3 github, curl it down in the make target, instead of embedding the script in the project...

Reply:
This file can be removed now...